lib/data-types.js

```diff
@@ -448,7 +448,7 @@ DATE.prototype.validate = function(value) {
 };
 
 DATE.prototype.$applyTimezone = function (date, options) {
-  date = moment(date);
+  date = moment.utc(date);
```

Comment:
As I remember, the date parameter here is a JS date object, so moment should be able to parse it correctly.

Can you show a test case?